examples/index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-unused-vars, react/jsx-no-undef, react/prop-types*/
```

Comment:
This file isn't part of a project but a repertoire of snippets exploring how Rollbar may be used.

So, these checks are bothersome, and the snippets are more explanatory if they look "as if" they were part of an actual larger project.

Reply:
I may end up adding this as a file comment, actually.